packages/language-server/src/ui5-model.ts

```diff
@@ -58,7 +59,17 @@ export async function getSemanticModelWithFetcher(
       if (apiJson === undefined) {
         getLogger().info("No cache found for UI5 lib", { libName });
         const url = baseUrl + libName.replace(/\./g, "/") + suffix;
-        const response = await fetcher(url);
+        const proxy = getProxy();
+        const fetchOptions = proxy
+          ? {
+              agent: new ProxyAgent(proxy, {
+                // Options, with all defaults
+                tunnel: true, // If true, will tunnel all HTTPS using CONNECT method
+                timeout: 5000, // Time in milli-seconds, to maximum wait for proxy connection to establish
```

Comment:
is 5000 a reasonable timeout? or too large?

Reply:
the 5000 was the default from the repo